napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
```

Comment:
Well, maybe not, that might be too confusing when there are `ObjectReference` and `FunctionReference` classes that use a different meaning of "reference".